src/applications/claims-status/actions/index.jsx

```diff
@@ -147,16 +148,51 @@ export function fetchClaimsSuccess(response) {
   };
 }
 
-export function getClaimsV2() {
+const POLLING_INTERVAL = 1000;
+
+export function pollClaimsStatus({
+  onFailure,
+  onSuccess,
+  pollingInterval,
+  request = apiRequest,
```

Comment:
This obviates the need to `mockFetch` which is tedious and pointless when we have a fetch helper that's already unit tested.